### ISSUE TITLE
Update to TF 2.0

### ISSUE THREAD

**Issue by mr-ubik** [google/compare_gan#28]
Will this project be updated to TF 2.0 once its release comes?

**Comment by Marvin182** [comment]
Yes, we will eventually update to TF 2.0. As of today TF 2.0 only supports training on TPU pods via TPUEstimator which removes much of the benefits of TF 2.0 (eager mode and distribution API). We will wait for TPUStrategy.

**Comment by heartInsert** [comment]
God ,  the   version's  problem  about  Tensorflow   is   the  world's   disaster
